pyproject.toml

```diff
@@ -44,7 +44,6 @@ dependencies = [
     "jupyter_server>=2.0.1,<3",
     "jupyterlab_server>=2.19.0,<3",
     "jupyter_ydoc>=0.3.0,<0.4.0",
```

Comment:
This has to be removed too as we bundle the JavaScript package we need but don't use the Python package.

```suggestion
```